dtls_debug.c

```diff
@@ -352,6 +374,43 @@ dtls_dsrv_hexdump_log(log_t level, const char *name, const unsigned char *buf, s
   if (maxlog < (int) level)
     return;
 
+  if (log_handler) {
```

Comment:
Hm, maybe, using a default handler would obsolete also some code then on other places.
Did you already thought of something similar?

Reply:
Not really (did not want the code to do `print_timestamp()` for what is passed to the alternative handler), but now you raise it, it makes sense and I can see how to do it.  I will have a look at reducing the code duplication.

Reply:
Alternatively, maybe we add a "compile switch" for those, who don't use it.

Reply:
Just pushed with code changed.  Not done the Contiki alternative as I have nothing to test code against.